iosApp/iosApp/Pages/StopDetails/DirectionPicker.swift

```diff
@@ -37,14 +37,21 @@ struct DirectionPicker: View {
                     let isSelected = filter?.directionId == direction
                     let action = { setFilter(.init(routeId: line?.id ?? route.id, directionId: direction)) }
 
-                    Button(action: action) {
+                    let button = Button(action: action) {
                         DirectionLabel(direction: directions[Int(direction)])
                             .padding(8)
                             .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .topLeading)
                     }
                     .background(isSelected ? Color(hex: route.color) : deselectedBackroundColor)
                     .foregroundStyle(isSelected ? Color(hex: route.textColor) : .deselectedToggleText)
                     .clipShape(.rect(cornerRadius: 6))
+
+                    if isSelected {
+                        button
+                            .accessibilityAddTraits(.isSelected)
```

Comment:
suggestion (non-blocking): Consider using the list version of `.accessibilityAddTraits` to simplify the conditional here:
```
.accessibilityAddTraits(isSelected ? [.isSelected] : [])

Reply:
88a61f19